test/unit/test_counter.dart

```diff
@@ -72,8 +75,9 @@ abstract class SubCounter implements Built<SubCounter, SubCounterBuilder> {
 // Middleware
 
 abstract class MiddlewareActions extends ReduxActions {
-  ActionDispatcher<int> doubleIt;
-  ActionDispatcher<int> tripleIt;
+  ActionDispatcher<void> doubleIt;
```

Comment:
This change is not actually related, but as the actions do not require any parameters, they can just be of tye `void` instead of `int`.